src/pages/dashboard/map-container.scss

```diff
@@ -66,10 +60,13 @@ $death-color: #404856;
   border: solid lightgray 2px;
   font-weight: initial;
 }
+#toggle-container{
+  width: 100%;
+}
 
 .map-toggle {
   cursor: pointer;
-  width: 100%;
+  float: left;
```

Comment:
I'd prefer to accomplish this with flexbox instead of floats

Reply:
Yeah, and also `float: left` breaks the layout of the other toggles, above the state charts further down the dashboard page.

Maybe try following how I did this with the other toggle. Keep a single parent `.map-toggle` as it was (with `width: 100%`). But add another class that sets `max-width` to avoid the container from stretching across its parent.

See:
- https://github.com/COVID19Tracking/website/blob/587c6622beb370976000386fd16f96f2e456e08c/src/pages/dashboard/dashboard.scss#L62-L66
- https://github.com/COVID19Tracking/website/blob/587c6622beb370976000386fd16f96f2e456e08c/src/pages/dashboard/_StateCumulativeTestsContainer.js#L164-L171

Reply:
Is that better practice than setting the element to display: inline-block?

Reply:
@AaronMullan well, ideally we'd have styles that are completely reusable. I actually didn't like having to set `max-width` for the other toggles, but did so because I didn't want to mess with the existing class above :P

Another option might be using the flexbox container better, as I think @goleary is recommending. If you remove your parent `.toggle-container` entirely and set this on `.map-toggle` instead of `display: inline-block`:

```
  align-self: flex-start;
```

It should left align in the parent `#state-map` container. But then you'll need to apply similar styling to the parent container of the other toggles and state charts below, [this div](https://github.com/COVID19Tracking/website/blob/587c6622beb370976000386fd16f96f2e456e08c/src/pages/dashboard/_StateCumulativeTestsContainer.js#L153), something like:

```
  display: flex;
  flex-direction: column;
```

Reply:
I was able to make the map one work with align-self: flex-start;  I don't see any repercussions below but I could refactor them to work the same way.